### ISSUE TITLE
call results in -32602 Invalid params when no params are given cause it sents null

### ISSUE THREAD

**Issue by basz** [elpheria/rpc-websockets#151]
Hello I'm integrating this library to connect to moonraker websockets. Now I find that moonrakers ws implementation is a little picky about null as params.

Sending a package as will work as expected.

```
{
    "jsonrpc": "2.0",
    "method": "server.info",
    "id": 2
}
```

however, this library will sent null if you do a call without params. Which for this method is what i need.

```
{
    "jsonrpc": "2.0",
    "method": "server.info",
    "params": null,
    "id": 2
}
```

Which result in a 

```{"code": -32602, "message": "Invalid params:"}```

I verified params: null is the issue as i changed this line to return undefined (meaning it will not get sent)

``` somewhere in lib
var message = {
          jsonrpc: "2.0",
          method: method,
          params: params || null,
          id: rpc_id
        };
```

Now my question. Can we change the null to undefined?

As it stands tis lib is not compatible with moonraker rpc json. 

ps. I also try to sent [] or {} as params.

**Comment by mkozjak** [comment]
Yes, based on the specification, the `params` key could be omitted. I'll see about implementing this.

**Comment by basz** [comment]
I can do a PR if you want

**Comment by mkozjak** [comment]
Fixed in #152
